scripts/fitTagAndProbe_script.py

```diff
@@ -79,11 +79,11 @@ def main(
         nparams = 6
         pdf_args.extend(
             [
-                "Voigtian::signal1Pass(m_vis, mean[91.2,85,95], width[2.495], sigma[2,1,4])",
```

Comment:
does the result improve, if the Z mass is less precise?

Reply:
I did not measured for that, since the majority of the fits done here have an pronounced Z-peak anyway. We could revert this change this back